jina/drivers/rank/aggregate/__init__.py

```diff
@@ -149,10 +149,6 @@ class Chunk2DocRankDriver(BaseAggregateMatchesRankerDriver):
                 |    |- matches: {granularity: k}
                 |
                 |-matches: {granularity: k-1} (Ranked according to Ranker Executor)
-
-        .. note::
```

Comment:
Why remove this. It's not correct?

Reply:
It changed to traverse on root.

Reply:
To my understanding, for this driver, we pass in the document, the document has several chunks, and chunks have matches. Then we traverse all the chunks of the document to do some ranking(score). Is it correct? 
If so, is it a traversal on chunks or root?
If not, what would be the case?

Reply:
This is correct, therefore the docs in input of the apply_all is the root

Reply:
😂  Ok. I think it's fine to remove it. The docstring above is clear and understandable to me.